skabase/SKABaseDevice/SKABaseDevice.xmi

```diff
@@ -34,6 +34,22 @@
       <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
       <DefaultPropValue>localhost</DefaultPropValue>
     </deviceProperties>
+    <deviceProperties name="CentralLogger" description="">
```

Comment:
What are the `CentralLogger` and `ElementLogger` properties for?

Reply:
These properties were earlier there, when we were setting them in code. They are not needed now. Will remove them. They got added by mistake during comparison.